### DISCUSSION TITLE
How to make arrow triangle base wider than its height?

### DISCUSSION THREAD

**Discussion by vitkhudenko** [skydoves/Balloon#216] [Q&A]
The issue I encountered has been introduced at the https://github.com/skydoves/Balloon/issues/64.

Despite the ticket is closed, I am still having the problem - it is impossible to make the arrow triangle base wider than its height.

This is what I am trying to achieve:

![image](https://user-images.githubusercontent.com/198951/127301046-50cba036-e792-4c6e-b869-a9fbd1c5ae19.png)

**Comment by skydoves** [comment]
Could you try using the setArrowTopPadding?

```kotlin
.setArrowSize(16)
.setArrowTopPadding(8) // or .setArrowBottomPadding(8) based on the orientation
```

**Reply by skydoves** [reply]
Another solution is using a custom arrow drawable that includes transparent padding on the image file.

**Reply by vitkhudenko** [reply]
@skydoves,  Will a custom arrow drawable work with the `setArrowColorMatchBalloon(true)`?

**Reply by skydoves** [reply]
Do your color of the balloon includes gradations?

**Reply by vitkhudenko** [reply]
@skydoves, yes, I use a color gradient for balloon background and that's why `setArrowColorMatchBalloon(true)` is a needed feature for my case.

**Reply by skydoves** [reply]
Great. Currently, the balloon uses normalized arrow size for computation of the entire size, but I will think about this more for supporting the different sizes (width & height).
